tock/utilization/unit.py

```diff
@@ -49,11 +52,11 @@ def _get_unit_billing_data(users, unit, recent_periods=None, fiscal_year=False):
         start, end, utilization = utilization_report(users, unit=unit)
 
     if utilization:
-        data = utilization.values('username', 'billable', 'target')
+        data = utilization.values('user_data', 'billable', 'target')
 
         # Build context for each employee
         output_data = [{
-                'username': employee['username'],
+                'display_name': UserData.objects.get(id=employee['user_data']).display_name,
```

Comment:
is adding this lookup here ok? i'm a little concerned about it if someone was using `values` to optimize the page as you mentioned @tbaxter-18f

Reply:
My concern is "Build context for each employee" I'm not sure how many employees we're talking about here, but that would be a DB lookup for each one.

This may be a thing we need to talk through more. @Jkrzy without diving in deep, it looks like we're keying the utilization table off of username. Thoughts?

Reply:
In the aggregation we're already traversing UserData -> User and grabbing the components for `display_name`, these queries would do that work again. I'd rather use those pieces we already have, and duplicate a bit of code, to generate `display_name`. 


```suggestion
                'display_name': f"{employee['first_name']} {employee['last_name']}".strip() or employee['username'],
```